mifosng-android/src/main/java/com/mifos/mifosxdroid/online/CentersActivity.java

```diff
@@ -17,18 +17,26 @@
 import com.mifos.mifosxdroid.online.grouplist.GroupListFragment;
 import com.mifos.mifosxdroid.online.savingsaccount.SavingsAccountFragment;
 import com.mifos.objects.client.Client;
+import com.mifos.utils.Constants;
 
 import java.util.List;
 
 public class CentersActivity extends MifosBaseActivity implements CenterListFragment
         .OnFragmentInteractionListener, GroupListFragment.OnFragmentInteractionListener {
 
+    private int centerId = 0;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_toolbar_container);
         showBackButton();
-        replaceFragment(new CenterListFragment(), false, R.id.container);
+        centerId = getIntent().getExtras().getInt(Constants.CENTER_ID);
```

Comment:
@therajanmaurya  This line is causing app crash as there is no extras if we navigate from dashboardactivity.java (home page from app).
Can I solve it by opening an issue and sending a Quick PR by putting a condition if getExtras() is not null.
Or do you want anything to do with changing in this pr?